base_old_migration_fields/readme/CONTRIBUTORS.md

```diff
@@ -0,0 +1,2 @@
+Ángel García de la Chica Herrera <angel.garcia@sygel.es>
+Alberto Martínez <alberto.martínez@sygel.es>
```

Comment:
use de new method please.

Reply:
change it to the following:

- [Sygel](https://www.sygel.es):

  - Ángel García de la Chica Herrera
  - Alberto Martínez